packages/fx-core/tests/common/local/localSettingsHelper.test.ts

```diff
@@ -73,6 +73,20 @@ describe("localSettingsHelper", () => {
       chai.assert.equal(localEnvs["AUTH_TAB_APP_ENDPOINT"], "https://localhost:53000");
     });
 
+    it("happy path without AAD plugin", async () => {
+      await fs.ensureDir(projectPath);
+      await fs.emptyDir(projectPath);
+
+      const projectSettingsAll = cloneDeep(projectSettings0);
+      delete projectSettingsAll.solutionSettings.activeResourcePlugins[0];
```

Comment:
can we find the index of aad plugin and remove it? this can avoid test failures when someone changes the content of activeResourcePlugins

Reply:
Fixed. Thanks!